BatteryMeasure.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BatteryManagement
+{
+    class BatteryMeasure
+    {
+        public float Temperature, StateOfCharge, ChargeRate;
+        public BatteryMeasure(float temperature, float soc, float chargeRate)
```

Comment:
How do we test BatteryMeasure methods ....?
why cann't these methods return status of check

Reply:
Now every check methods will return bool values, represents the status of the check. Henceforth, we can test the BatteryMeasure Methods.